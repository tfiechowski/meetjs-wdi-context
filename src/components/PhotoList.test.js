import { shallow } from "enzyme";
import React from "react";
import PhotosList from "./PhotoList";

describe("Components", () => {
  describe("PhotosList", () => {
    it("Renders the list", () => {
      const photos = [
        { id: 37, title: "Back in Black" },
        { id: 42, title: "Highway to Hell" }
      ];

      const wrapper = shallow(<PhotosList photos={photos} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
