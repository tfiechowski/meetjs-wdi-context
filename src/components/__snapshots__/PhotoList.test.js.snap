// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components PhotosList Renders the list 1`] = `
<div>
  <div
    className="photo-dark"
    key="37"
  >
    Back in Black
  </div>
  <div
    className="photo-dark"
    key="42"
  >
    Highway to Hell
  </div>
  <button>
    Fetch more photos
  </button>
</div>
`;
